08-Secure by default/Secure Templates/ARM/Synapse.json

```diff
@@ -0,0 +1,113 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "synapseWorkspaceName":{
+            "type": "string",
+            "metadata": {
+                "description": "Synapse workspace name"
+            }
+        },
+        "dataLakeStorageAccountUrl":{
+            "type": "string",
+            "metadata":{
+                "description": "url of the data lake storage account which we need to associate with synapse workspace"
+            }
+        },
+        "dataLakeStorageFileSystem":{
+            "type": "string",
+            "metadata":{
+                "description": "File system name to associate with data lake storage and synapse workspace"
+            }
+        },
+        "sqlAdminUser":{
+            "type": "string",
+            "metadata":{
+                "description": "sql admin user "
+            }
+        },
+        "sqlAdminUserPassword":{
+            "type": "string",
+            "metadata":{
+                "description": "Password for sql admin user."
+            }
+        },
+        "storageAccountId":{
+            "type": "string",
+            "metadata":{
+                "description": "Storage account id to which the diagnostic logs and metrices will be sent."
+            }
+        }
+    },
+    "functions": [],
+    "variables": {},
+    "resources": [
+        {
+            "type": "Microsoft.Synapse/workspaces",
+            "apiVersion": "2019-06-01-preview",
+            "name": "[parameters('synapseWorkspaceName')]",
+            "location": "[resourceGroup().location]",
+            "identity": {
+                "type": "SystemAssigned"
+            },
+            "properties": {
+                "defaultDataLakeStorage": {
+                    "accountUrl": "[parameters('dataLakeStorageAccountUrl')]",
+                    "filesystem": "[parameters('dataLakeStorageFileSystem')]"
+                },
+                "sqlAdministratorLogin": "[parameters('sqlAdminUser')]",
+                "sqlAdministratorLoginPassword": "[parameters('sqlAdminUserPassword')]"
+            }
+        },
+        {
+            // This one is required for diagnostic settings. (Azure_SynapseWorkspace_Audit_Enable_DiagnosticSettings_Trial)
```

Comment:
NIT: Please remove text: This one is required for diagnostic settings

Just keep control Id and display name like / Azure_ServiceFabric_DP_Set_Property_ClusterProtectionLevel - The ClusterProtectionLevel property must be set to "EncryptAndSign".

Reply:
Fixed.